template.yaml

```diff
@@ -192,6 +192,9 @@ Resources:
       Layers:
         - !Ref CommonLambdaLayer
       Timeout: 900
+      Architectures:
```

Comment:
@sakthishanmugam  Were you able to test this ? Also can we check if Graviton Architecture is available in most regions else we need to probably specify this in docs somewhere